libswift/Sources/Optimizer/PassManager/PassUtils.swift

```diff
@@ -70,25 +70,26 @@ struct FunctionPass {
   }
 
   func run(_ bridgedCtxt: BridgedFunctionPassCtxt) {
-    let function = bridgedCtxt.function.function
+    let function = bridgedCtxt.function
     let context = PassContext(passContext: bridgedCtxt.passContext)
     runFunction(function, context)
   }
 }
 
-struct InstructionPass<InstType: Instruction> {
```

Comment:
Woops, this is one of the rough edges: I can revert this change. Generics are working fine (as you can see with `List`).